### PR TITLE
Feature roles on user model

### PR DESCRIPTION
**Pull request by MagnificRogue**

### CODE REVIEW

File: tests/Unit/UserTest.php [hunk 1]
```diff
@@ -45,7 +45,7 @@ public function testCreateUser(){
     $users = json_decode($response->content(),true)["data"]["users"];
 
     //create a new user 
-    $response = $this->callAuthenticated('GET', '/users/create',["name"=>"Brian Ludwig, JR","password"=>"Harper86","email"=>"faker@poop.com"]);
+    $response = $this->callAuthenticated('GET', '/users/create',["name"=>"Brian Ludwig, JR","password"=>"Harper86","email"=>"faker@poop.com", "permission" => "standard"]);
```
Comment (hcbl1212) [inline tests/Unit/UserTest.php]: Maybe a test that shows a standard user can't create a user?

Comment (MagnificRogue) [inline tests/Unit/UserTest.php]: That is outside the scope of this feature. That will be handled when I have guards and policies implemented.

Comment (hcbl1212) [inline tests/Unit/UserTest.php]: sounds good!
